v22.1/map-sql-activity-to-app.md

```diff
@@ -0,0 +1,57 @@
+
+You can also programmatically filter `crdb_internal` tables [`crdb_internal.statement_statistics`](crdb-internal.html#statement_statistics) and [`crdb_internal.transaction_statistics`](crdb-internal.html#transaction_statistics) by application name. This example shows the first 60 characters of query text and statement statistics for queries associated with the `movr_app` application:
+
+{% include copy-clipboard.html %}
+~~~sql
+SELECT app_name, substring(metadata ->> 'query',1,60) AS statement_text,
+   metadata -> 'distsql' AS is_distsql,
+   metadata -> 'fullScan' AS has_full_scan,
+   metadata -> 'vec' AS used_vec,
+   statistics -> 'execution_statistics' -> 'contentionTime' -> 'mean' AS contention_time_mean,
+   statistics -> 'statistics' -> 'cnt' AS execution_count,
+   statistics -> 'statistics' -> 'firstAttemptCnt' AS num_first_attempts,
+   statistics -> 'statistics' -> 'numRows' -> 'mean' AS num_rows_returned_mean,
+   statistics -> 'statistics' -> 'rowsRead' -> 'mean' AS num_rows_read_mean,
+   statistics -> 'statistics' -> 'runLat' -> 'mean' AS runtime_latency_mean
+FROM movr.crdb_internal.statement_statistics
+WHERE app_name = 'movr_app';
+~~~
+
+~~~
+  app_name |                        statement_text                        | is_distsql | has_full_scan | used_vec | contention_time_mean | execution_count | num_first_attempts | num_rows_returned_mean | num_rows_read_mean | runtime_latency_mean
+-----------+--------------------------------------------------------------+------------+---------------+----------+----------------------+-----------------+--------------------+------------------------+--------------------+------------------------
+  movr_app | SELECT * FROM users                                          | true       | true          | true     |                    0 |               1 |                  1 | 5E+1                   | 5E+1               |              0.002202
+  movr_app | SELECT name, count(rides.id) AS sum FROM users JOIN rides ON | true       | true          | true     |                    0 |               1 |                  1 | 1E+1                   | 5.5E+2             |              0.005338
+...
+~~~
+
+## Trace SQL activity using metrics
+
+Often signals to problems start at high-level metrics. The [SQL dashboard](ui-sql-dashboard.html) has metrics for [SQL sessions](ui-sql-dashboard.html#open-sql-sessions), [SQL statements](ui-sql-dashboard.html#active-sql-statements), [SQL statement errors](ui-sql-dashboard.html#sql-statement-errors), etc., that can quickly signify potential issues with your application.
+
+The statement and transaction statistics described in the preceding section enable you to correlate high-level metrics with lower level SQL activity information where you can identify the specific statements and transactions that were running during that time.
```

Comment:
Suggest adding a "See also" section below this that links to other related pages on optimization and troubleshooting

https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#see-also